cli/cmd/sync.go

```diff
@@ -36,7 +36,27 @@ func NewCmdSync() *cobra.Command {
 	return cmd
 }
 
+func getCQDir(cmd *cobra.Command) (string, error) {
+	// backwards compatibility
+	cqDir, err := cmd.Flags().GetString("data-dir") // old flag
```

Comment:
We are taking our backward compatability too far. This never worked. no need to make it work now as we deprecating it anyway - otherwise we just introduce more code to maintain for no reason.

Reply:
Once this is remove I think no need for this function anymore

Reply:
Done in https://github.com/cloudquery/cloudquery/pull/2796